src/stores/Token.ts

```diff
@@ -144,6 +146,10 @@ export default class TokenStore {
                             payload.allowance,
                             payload.lastFetched
                         );
+
```

Comment:
I would put this logic in the component rather than here - this also would set the enabled property from any token fetch. I'm fine merging and changing later.